src/dxtbx/model/detector_helpers.py

```diff
@@ -294,9 +294,15 @@ def set_detector_distance(detector, distance):
     detector[0].set_frame(fast_axis, slow_axis, origin)
 
 
-def project_2d(detector):
-    """
-    Project panel origin, fast and slow onto the best-fitting 2D plane.
+def get_detector_projection_2d_axes(detector):
+
+    """Project panel origins, fast and slow axes onto the best-fitting 2D plane.
+
+    :param detector: detector used to generate axes
+    :type detector: Detector
+
+    :return: 2D origin, fast, and slow vectors for each panel
+    :rtype: list(tuple(float, float)), list(tuple(float, float)), list(tuple(float, float))
```

Comment:
As below, I think type annotations would be more modern/useful here than this style of docstrings.